.github/workflows/sbt-node-snyk-pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Simple Snyk test for SBT + Node
+
+on:
+  workflow_call:
+    inputs:
+      DEBUG:
+        type: string
+        required: false
+      ORG:
+        type: string
+        required: true
+      JAVA_VERSION:
+        type: string
+        required: false
+        default: "11"
+      NODE_VERSION_FILE:
+        type: string
+        required: false
+        default: ".nvmrc"
+    secrets:
+      SNYK_TOKEN:
+        required: true
+
+jobs:
+    security:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout branch
+              uses: actions/checkout@v2
+
+            - uses: snyk/actions/setup@0.3.0
+            - uses: actions/setup-node@v2
+              with:
+                  node-version-file: ${{ inputs.NODE_VERSION_FILE }}
+
+            - uses: actions/setup-java@v2
+              with:
+                  java-version: ${{ inputs.JAVA_VERSION }}
+                  distribution: "adopt"
+
+            - name: Snyk test
+              run: echo ${INPUT_DEBUG} ${INPUT_DEBUG:+ -d} && snyk test ${INPUT_DEBUG:+ -d} --severity-threshold=critical --all-projects --org="${{ inputs.ORG }}"
```

Comment:
Should we do high severity as well? Or maybe better to make it configurable actually?